open-metadata-conformance-suite/open-metadata-conformance-suite-spring/src/main/java/org/odpi/openmetadata/conformance/server/spring/ConformanceSuiteResource.java

```diff
@@ -12,6 +12,7 @@
 import org.springframework.web.bind.annotation.RequestMapping;
```

Comment:
I've not tested, but I wonder if we should remove the RequestMapping - ie clean up the imports, since we're changing to the get/put etc mappings?

Reply:
So this fix only removes the the Get's so the RequestMapping are needed for the other HTTP verbs. Post, Put etc. When I remove all those the last PR will remove the 
import org.springframework.web.bind.annotation.RequestMapping;
when it's no longer needed ?

Reply:
perfect! thanks for clarifying!